assets-lfs/README.md

```diff
@@ -0,0 +1,16 @@
+# Assets in LFS
+
+Assets under this directory are stored in Git LFS.
+
+
+To fetch files from LFS, install `git-lfs` first:
+
+```
+brew install git-lfs
```

Comment:
How about adding instruction for Linux as well?

Reply:
Done